src/managers/views/iframe.js

```diff
@@ -146,20 +146,23 @@ class IframeView {
 				// apply the layout function to the contents
 				this.settings.layout.format(this.contents);
 
-				// Expand the iframe to the full size of the content
-				this.expand();
-
-				// Listen for events that require an expansion of the iframe
-				this.addListeners();
+        setTimeout((function() {
```

Comment:
Hi, I'm sorry for the delay. I was on holidays.

Here you have a test book. If you load the book using the v0.2, it will work ok, but if you load with v0.3, you'll see the pages starting to be displaced and you'll have to click "next" several times in order to reach the next html.

https://drive.google.com/file/d/0B3leacKmz55oMHlyUGxsc3ctZEE/view?usp=sharing